code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -194,8 +194,7 @@
 	heat_reagents(seconds_per_tick)
 
 	//send updates to ui. faster than SStgui.update_uis
-	for(var/datum/tgui/ui in src.open_uis)
-		ui.send_update()
```

Comment:
I. Don't really know why this is relevant to the pr, should just be a separate pr

Reply:
this is irrelevant anyway, will remove it